pkg/steampipeconfig/modconfig/parsed_name.go

```diff
@@ -54,6 +54,7 @@ func (p *ParsedResourceName) ToFullNameWithMod(mod string) string {
 	return BuildFullResourceName(mod, p.ItemType, p.Name)
 }
 
+// BuildFullResourceName generates a Fully Qualified name from the given components
```

Comment:
no need for caps  -> `fully qualified`

give example

Reply:
either include validation here or raise issue for validation and put a TODO here referencing issue